clang/test/SemaSYCL/esimd_widen.cpp

```diff
@@ -0,0 +1,28 @@
+// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify %s
```

Comment:
Can you add a check for the host case too?

Reply:
Done